### ISSUE TITLE
api error: cannot open data file -0.5086417 

### ISSUE THREAD

**Issue by kswada** [Microsoft/LightGBM#477]
## Environment info
Operating System:  Windows 10
CPU:
C++/Python/R version:  R x64 3.3.1

I installed by
devtools::install_github("Microsoft/LightGBM", subdir = "R-package")

and I run the scrpits below.
------------------
library(lightgbm)
data(agaricus.train, package='lightgbm')
train <- agaricus.train
dtrain <- lgb.Dataset(train$data, label=train$label)
params <- list(objective="regression", metric="l2")
model <- lgb.cv(params, dtrain, 10, nfold=5, min_data=1, learning_rate=1, early_stopping_rounds=10)
------------------

but shows the message...

lgb.call("LGBM_DatasetCreateFromCSC_R", ret = handle, private$raw_data@p,  でエラー: 
  api error: cannot open data file -0.5086417
Please advise...

**Comment by guolinke** [comment]
@Laurae2  need the ```lgb.unloader()``` ?

**Comment by kswada** [comment]
Oh.. Thanks, it works by "lgb.unloader()".
